docs/apache-airflow/tutorial/fundamentals.rst

```diff
@@ -694,7 +403,7 @@ Here's a few things you might want to do next:
     - Take an in-depth tour of the UI - click all the things!
     - Keep reading the docs!
 
-      - Review the :doc:`how-to guides<howto/index>`, which include a guide to writing your own operator
+      - Review the :doc:`how-to guides</howto/index>`, which include a guide to writing your own operator
       - Review the :ref:`Command Line Interface Reference<cli>`
       - Review the :ref:`List of operators <pythonapi:operators>`
       - Review the :ref:`Macros reference<macros>`
```

Comment:
I agree. And I also think that the order of the tutorials should be flipped, so that the taskflow one is next instead of the dockerized pipeline. I'll add a line here encouraging folks to "take the next step by learning taskflow".

Reply:
I've now added the line 👍